azurerm/internal/services/datafactory/data_factory_linked_service_synapse_resource_test.go

```diff
@@ -0,0 +1,221 @@
+package datafactory_test
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMDataFactoryLinkedServiceSynapse_ConnectionString(t *testing.T) {
```

Comment:
could we remove AzureRM from the test names?
```suggestion
func TestAccDataFactoryLinkedServiceSynapse_ConnectionString(t *testing.T) {
```